### ISSUE TITLE
Show a spinner when the ImageControl is loading the image

### ISSUE THREAD

**Issue by tomjn** [humanmade/block-editor-components#19]
When you pick an image, `imageUrl` may not have a value. This could be due to an error or because it hasn't loaded in yet, but during this period the only indication it worked is the button changes its text.

Instead this `&&` check should be a ternary or another check that displays the WP loading spinner to give user feedback:

```jsx
							{ imageUrl && (
								<Button isLink onClick={ open }>
									<img alt="" src={ imageUrl } />
								</Button>
							) }
```

**Comment by tomjn** [comment]
Theoretically fixed in #20

**Comment by tfrommen** [comment]
Implemented in #21.
